<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VeroMaaritys</class>
 <widget class="QWidget" name="VeroMaaritys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tuloverojen viite</string>
       </property>
       <property name="buddy">
        <cstring>tuloveroviiteEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="tuloveroviiteEdit">
       <property name="Asetus" stdset="0">
        <string>VeroTuloViite</string>
       </property>
       <property name="Validator" stdset="0">
        <string notr="true">Viite</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Oma-aloitteisten verojen viite</string>
       </property>
       <property name="buddy">
        <cstring>omaehtoistenviiteEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="omaehtoistenviiteEdit">
       <property name="Asetus" stdset="0">
        <string notr="true">VeroOmaViite</string>
       </property>
       <property name="Validator" stdset="0">
        <string notr="true">Viite</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Arvonlisäveron kirjaus arvonlisäveroilmoituksen yhteydessä</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>Kirjaa aina Verovelat-tilille</string>
        </property>
        <property name="Asetus" stdset="0">
         <string notr="true">AlvSuoraanVerovelkaan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="saatavaRadio">
        <property name="text">
         <string>Kirjaa maksettavat Verovelat-tilille ja palautukset Verosaatavat-tilille</string>
        </property>
        <property name="Asetus" stdset="0">
         <string notr="true">AlvPalautusSaatavatilille</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="maksutili">
        <property name="text">
         <string>Kirjaa Maksettava alv/Palautettava alv -tileille</string>
        </property>
        <property name="Asetus" stdset="0">
         <string notr="true">AlvMaksutilinKautta</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Yhteyshenkilö veroja ilmoitettaessa</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_2</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="Asetus" stdset="0">
        <string notr="true">VeroYhteysHenkilo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Yhteyspuhelinnumero veroja ilmoitettaessa</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="Asetus" stdset="0">
        <string notr="true">VeroYhteysPuhelin</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="varmenneGroup">
     <property name="title">
      <string>Verohallinnon varmenne</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
      <item>
       <widget class="QLabel" name="varmenneInfo">
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="uusiVarmenneNappi">
          <property name="text">
           <string>Uusi varmenne</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/lisaa.png</normaloff>:/pic/lisaa.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="poistaVarmenneNappi">
          <property name="text">
           <string>Poista varmenne</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/poista.png</normaloff>:/pic/poista.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>229</width>
       <height>197</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
